docs/notebooks/icnn_inits.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>if this is only used for plots, maybe introduce right before plots?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>See comment above.</p>